examples/Console/TestOtlpExporter.cs

```diff
@@ -16,14 +16,15 @@
 
 using System;
 using OpenTelemetry;
+using OpenTelemetry.Exporter;
 using OpenTelemetry.Resources;
 using OpenTelemetry.Trace;
 
 namespace Examples.Console
 {
     internal static class TestOtlpExporter
     {
-        internal static object Run(string endpoint)
+        internal static object Run(string endpoint, ExportProtocol protocol = ExportProtocol.Grpc)
```

Comment:
all existing examples only expose simple types to be parsed from command line.. We may want to stick with the same approach.. (I don't really recollect if we intentionally used basic types to avoid any parsing issues..)

Reply:
If we use here basic type then we shift error to later stage where string value is mapped to the enum in case of unsupported value. But we can stick with string type if needed.

Reply:
Switched protocol option to string type in test console app.